tracer/test/snapshots/AwsSqsTests.NetCore.SchemaV0.verified.txt

```diff
@@ -34,11 +34,13 @@
       aws.queue.url: http://localhost:00000/000000000000/MyAsyncSQSQueue,
       aws.requestId: Guid_Empty,
       aws.service: SQS,
+      aws_service: SQS,
```

Comment:
I've already got prs in other tracers merged where we create these duplicate tags. I'll dm you a link to the doc explaining why we did this but here is a brief explanation

In order to achieve pivoting from metrics to traces to work out of the box we need these new tags. For instance if we wish to pivot from aws integration metric `aws.dynamodb.table_size` then our spans metadata needs to match with metrics metadata. This also allows us to unlock benefit of inferring related resources on the serverless function page.

<img width="1007" alt="Screen Shot 2023-04-27 at 8 07 07 PM" src="https://user-images.githubusercontent.com/49878080/235045306-5bfc816a-d437-472c-b901-c8e5abe41564.png">


Why not remove old tags such as aws.region, or aws.sqs.queue_name now? Many of these tags are already used by customer monitors, dashboards, metrics, notebooks... Therefore I have logged a jira ticket under framework integrations to eventually phase these old tags out.

If the aws integrations metrics could change their metric tag values then this would solve the pivot issue but changing a tag on a metric like that is a much bigger breaking change as much more customer resources rely on those and their (metrics) are stored for much longer to my knowledge.

Reply:
The motivations make sense. My concern here is about being mindful of how many bytes we use for each span, so if there's a way to avoid adding the tags here (which will go on the network) and adding them on the backend, that would be even better